overrides/components/theme-card.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "$veda-ui/react";
```

Comment:
This component is meant to be shared across landing page and theme landing page @sandrahoang686

Reply:
Can we change the file name to `theme-cards` since this component returns a list of theme-cards and not a single theme-card